app/connectors/des/DesConnector.scala

```diff
@@ -73,11 +73,11 @@ class DesConnector @Inject() (servicesConfig: ServicesConfig, httpClient: HttpCl
     httpClient.GET[Option[DirectDebitData]](getDDUrl)
   }
 
-  def getRepaymentDetails(vrn: Vrn): Future[Seq[RepaymentDetailData]] = {
+  def getRepaymentDetails(vrn: Vrn): Future[Option[Seq[RepaymentDetailData]]] = {
```

Comment:
If there is no difference between empty list and 404 we'd prefer to return empty list instead of 404.